README.md

```diff
@@ -9,6 +9,33 @@
 Firmware for the [CyphalPicoBase/CAN](https://github.com/generationmake/CyphalPicoBase-CAN) board. You can buy one [here](https://cyphal.store/products/lxrobotics-cyphalpicobase-can) 💸.
 
 ## How-to-build/upload
+
+This firmware relies on the excellent work of Earle F. Philhower, III to create an arduino integration with the Raspberry Pi Pico SDK (as opposed to the Mbed-based integration).
+You'll need to install this first to build. Instructions can be found [in the arduino-pico documentation](https://arduino-pico.readthedocs.io/en/latest/install.html#installing-via-arduino-cli) but the TLDR is:
+
+```bash
+arduino-cli config add board_manager.additional_urls https://github.com/earlephilhower/arduino-pico/releases/download/global/package_rp2040_index.json
+arduino-cli core update-index
+arduino-cli core install rp2040:rp2040
+```
+
+Next you'll need to install the libraries this firmware depends on. There are several public libraries:
+
+```bash
+arduino-cli lib update-index
+arduino-cli lib install 107-Arduino-Debug 107-Arduino-MCP2515 107-Arduino-UniqueId 107-Arduino-24LCxx 107-Arduino-littlefs
+arduino-cli lib install "Adafruit NeoPixel"
+```
+
+And two private ones. For these you'll need to [enable installing from git](https://arduino.github.io/arduino-cli/1.0/configuration/#configuration-keys):
+
+```bash
+export ARDUINO_LIBRARY_ENABLE_UNSAFE_INSTALL=true
+arduino-cli lib install --git-url https://github.com/107-systems/107-Arduino-Cyphal.git
```

Comment:
Those libraries are part of the official library manager so `arduino-cli lib install 107-Arduino-Cyphal` should suffice :wink: .

Reply:
I'll give it a try again. I thought this one failed.

Reply:
See here @thirtytwobits : https://github.com/arduino/library-registry/blob/main/repositories.txt#L323-L324 .

Reply:
Sorry, ran out of time last night. WIll update this PR today.